packages/lumx-react/src/components/comment-block/CommentBlock.tsx

```diff
@@ -33,7 +35,7 @@ interface CommentBlockProps extends GenericProps {
     /* Username display */
     name: string;
     /* Content to be displayed */
-    text: HTMLElement | string;
+    text: HTMLElement | ReactNode | string;
```

Comment:
Can you remove `HTMLElement` everywhere. It's an invalid type. We can't actually give DOM element as props here ^^

Reply:
Haha I actually started to do that and then thought it maybe wasn't the right time to do that.
But sure thing, I'll remove it from this component.